### PR TITLE
update nav, add more abbrevations, fix links 

### PR DESCRIPTION
**Pull request by VladoLavor**

### CODE REVIEW

File: docs/plugins/db-plugins.md [hunk 1]
```diff
@@ -307,3 +307,6 @@ Data will not be propagated if target path directory.
 [datasync-image]: ../img/user-guide/datasync_watch.png
 [datasync-publish-image]: ../img/user-guide/datasync_pub.png
 [db-image]: ../img/user-guide/db.png
+```
```
Comment (ondrej-fabry) [inline docs/plugins/db-plugins.md]: Typo?

File: docs/tutorials/07_vpp-connection.md [hunk 2]
```diff
@@ -389,5 +389,7 @@ func main() {
 
 The output of this call will be shown in the log as a repeated message that the VPP interface with a given index was received. In the multi-request, the reply message usually contains several fields where some of them are VPP specific (like interface admin status, default MTU, internal names, etc.).
 
-[1]: ../plugins/core-vpp-plugins.md#govppmux-plugin
+[1]: ../plugins/vpp-plugins.md#govppmux-plugin
 [2]: https://wiki.fd.io/view/GoVPP
+
+*[CLI]: Command Language Interpreter
```
Comment (ondrej-fabry) [inline docs/tutorials/07_vpp-connection.md]: CLI stands for `Command Line Interface`

File: docs/plugins/db-plugins.md (hunk 1 above)
Comment (VladoLavor) [reply docs/plugins/db-plugins.md]: Fixed, the top marker was ignored

File: docs/user-guide/quickstart.md [hunk 3]
```diff
@@ -224,3 +224,6 @@ $ docker exec -it vpp-agent vppctl -s localhost:5002 show bridge-domain
 [docker-install]: https://docs.docker.com/cs-engine/1.13/
 [dockerhub]: https://hub.docker.com/u/ligato
 [postman-install]: https://learning.getpostman.com/docs/postman/launching_postman/installation_and_updates/
+
+*[CLI]: Command Language Interpreter
```
Comment (ondrej-fabry) [inline docs/user-guide/quickstart.md]: Ditto.

**Pull request merged**